### PR TITLE
[PORT] Improved Traffic System (with Updates!)

### PR DESCRIPTION
**Pull request by KillianKirilenko**
Months ago, I rewrote parts of Citadel's "busy space" module, in order to allow for a much greater variance in traffic events and to strip out certain content. The following changes have been made to adapt it for use here;
* Moved entire system to an independent module folder, so that upstream changes won't overwrite anything.
* Removed three more Virgo-specific factions.
* Removed the Citadel-specific factions; Antares & BlackStar.

The following additional changes were made;
* Renamed the Traveller's Guide and updated the Pocket Newscaster description.
* Changed the system name in the map defines to match lore given [here](https://www.yawn.ocry.com/index.php?title=Cryogaia), as it's called by the events system.
* Added the Teshari Expeditionary Fleet, SAARE, Smuggler, Pirate, and Vox Marauder factions.
* Added several new events based around smugglers, piracy, and the marauders.
* Adjusted several existing events with numerous minor grammatical/etc. fixes.
* Updated many TSC/etc. descriptions used in the Traveller's Guide.
* And a bunch of other minor tweaks like event frequency.

As with my original PRs, this has been run through the wringer on local testing, both to fix runtimes/errors and actual oddities in the events themselves.

The old traffic system components are retained for reference, but removed from the .dme.

### CODE REVIEW

File: code/modules/lore_codex/codex.dm [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
 /obj/item/weapon/book/codex/lore/news
 	name = "Daedalus Pocket Newscaster"
-	desc = "A regularly-updating compendium of articles on current events. Essential for new arrivals in the Vir system and anyone interested in politics."
+	desc = "A regularly-updating compendium of articles on current events. Essential for new arrivals in the Borealis system and anyone interested in politics."
```
Comment (Shadow-Quill) [inline code/modules/lore_codex/codex.dm]: ditto

File: code/modules/lore_codex/codex.dm [hunk 2]
```diff
@@ -19,8 +19,8 @@
 	tree.display(user)
 
 /obj/item/weapon/book/codex/lore/vir // Vorestation Edits throughout this object.
-	name = "The Traveler's Guide to Human Space: Virgo-Erigone Edition"
-	desc = "Contains useful information about the world around you.  It seems to have been written for travelers to Virgo-Erigone, human or not. It also \
+	name = "The Traveler's Guide to Human Space: Borealis Edition"
+	desc = "Contains useful information about the world around you.  It seems to have been written for travelers to the Borealis system, human or not. It also \
```
Comment (Shadow-Quill) [inline code/modules/lore_codex/codex.dm]: ditto

File: code/modules/lore_codex/codex.dm [hunk 3]
```diff
@@ -1,7 +1,7 @@
 // Inherits from /book/ so it can fit on bookshelves.
 /obj/item/weapon/book/codex // Vorestation Edits throughout this object.
-	name = "The Traveler's Guide to Human Space: Virgo-Erigone Edition"
-	desc = "Contains useful information about the world around you.  It seems to have been written for travelers to Virgo-Erigone, human or not. It also \
+	name = "The Traveler's Guide to Human Space: Borealis Edition"
+	desc = "Contains useful information about the world around you.  It seems to have been written for travelers to the Borealis system, human or not. It also \
```
Comment (Shadow-Quill) [inline code/modules/lore_codex/codex.dm]: ditto

File: code/modules/client/preference_setup/loadout/loadout_utility.dm [hunk 4]
```diff
@@ -31,7 +31,7 @@
 	path = /obj/item/device/camera
 
 /datum/gear/utility/codex
-	display_name = "the traveler's guide to vir"
+	display_name = "the traveler's guide to borealis"
```
Comment (Shadow-Quill) [inline code/modules/client/preference_setup/loadout/loadout_utility.dm]: ditto

**Comment by KillianKirilenko** [comment]
Not sure if the desc edits are tagged quite right but those wrapped lines are a pain in the ass.

**Comment by Shadow-Quill** [comment]
That should be fine. Don't forget to smack the "Update branch" button around here
image

**Comment by cebutris** [comment]
Really enjoyed how this was implemented at Cit, and I'm excited to see it here!

**Comment by KillianKirilenko** [comment]
I'm glad to hear it! ~~And I look forward to hearing more stories of people freaking out.~~

Here's some bonus stats/info, whilst I'm still elbow-deep in the code;
> Factions; (25)
NT, Hephaestus, Vey-Med, Zeng-Hu, Ward-Takahashi, Bishop, Morpheus, Xion, FTU, Major Bill's, Grayson, Aether, Focal Point, StarFlight Inc., Independents, SDF, Smugglers, Pirates, Marauders, SolGov/USG, Teshari Expeditionary Fleet, USDF, PCRC, HIVE, SAARE
> 
> Events; (20)
Mayday, Police Scan (variant: Flee!), SDF Scan (variant: Flee!), SDF Engaged, DefCon: Hostile Sighted!, Distress Call: Under Attack!, Travel Advisory, Navigation Warning, Docking Request (variants; Resupply, Repair, Security, Medical Crisis), Undock Request (variants; Denied, Delayed), SDF Beginning Patrol, General Request
> 
> Smugglers only appear for the *Flee!* scan event variants, and Pirates & Marauders only appear for the *SDF Engaged*, *DefCon: Hostile Sighted!* and *Distress Call: Under Attack!* events.

**Comment by Repede** [comment]
Yo, I Travis has marked you as a 'abuser' or whatever, you may have to contact them at https://travis-ci.com/ to get that tag lifted, otherwise the PR Bot won't actually run checks. Sorry!

**Comment by KillianKirilenko** [comment]
I've been submitting PRs to Citadel since April last year without receiving any complaints from Travis besides occasional compile issues with Active Edges, so I'd love to know why this has suddenly become a problem. It seems to be a theme with this repo specifically, if anything.

Oh well. This'll give me a little more time to test and finetune a couple new events I'm working on.

**Comment by izac112** [comment]
This happend to someone else too apparently the guys over at travis did a ban wave you can just send an email to them and it get lifted without any problems, Doubt they tell you why besides our system did it or something.

**Comment by KillianKirilenko** [comment]
Yeah I already sent an email and got it cleared up, I've just been tweaking and testing locally in the meantime. Got a couple more minor events lined up for the SDF (ending patrol & patrol status update).

**Comment by Repede** [comment]
I should ask the Travis team if there's some issue with our repo because I think it's like the 4th person that has had that issue recently with us/Travis.

**Comment by KillianKirilenko** [comment]
Yeah, that was why I said it seemed to be a theme with this repo. Perhaps it's something to do with the similarity, like it's trying to catch people who might be saturating multiple downstreams with unnecessary PRs? 🤔

Anyway, this PR's all set and done for now.

**Pull request merged**